test/atlas/domain/sequence_diagram_test.clj

```diff
@@ -0,0 +1,104 @@
+(ns atlas.domain.sequence-diagram-test
+  (:require [atlas.domain.sequence-diagram :as nut]
+            [clojure.test :refer [is testing]]
+            [common-clj.clojure-test-helpers.core :refer [deftest]]))
+
+(def trace
+  {:trace-id "1"
+   :spans    [{:trace-id       "1"
+               :span-id        "1"
+               :process-id     :p1
+               :operation-name "http.in GET /api/orders/1"
+               :start-time     1500000000000000
+               :duration       1000
+               :references     []
+               :tags          [{:key   "span.kind"
+                                :type  "string"
+                                :value "server"}
+                               {:key   "http.method"
+                                :type  "string"
+                                :value "GET"}
+                               {:key   "http.url"
+                                :type  "string"
+                                :value "/api/orders/1"}]}
+
+              {:trace-id       "1"
+               :span-id        "2"
+               :process-id     :p1
+               :operation-name "http.out GET /api/orders/1"
+               :start-time     1500000000100000
+               :duration       300
+               :references     [{:ref-type :child-of
+                                 :trace-id "1"
+                                 :span-id  "1"}]
+               :tags          [{:key   "span.kind"
+                                :type  "string"
+                                :value "client"}
+                               {:key   "http.method"
+                                :type  "string"
+                                :value "GET"}
+                               {:key   "http.url"
+                                :type  "string"
+                                :value "/api/orders/1"}]}
+
+              {:trace-id       "1"
+               :span-id        "3"
+               :process-id     :p2
+               :operation-name "http.in GET /api/orders/1"
+               :start-time     1500000000200000
+               :duration       100
+               :references     [{:ref-type :child-of
+                                 :trace-id "1"
+                                 :span-id  "2"}]
+               :tags          [{:key   "span.kind"
+                                :type  "string"
+                                :value "server"}
+                               {:key   "http.method"
+                                :type  "string"
+                                :value "GET"}
+                               {:key   "http.url"
+                                :type  "string"
+                                :value "/api/orders/1"}]}]
+
+   :processes {:p1 {:service-name "bff"}
+               :p2 {:service-name "orders"}}})
+
+(deftest start-time
+  (testing "builds start-time from trace"
+    (is (= #epoch 1500000000000
```

Comment:
I think this 1500000000000 should be extracted to a const (and then used here and as the `:start-time` of the first span); it could be called `FIRST_SPAN_START_TIME`? (for the same reason we've already discussed in the other PR)